docs/supported-components.human

```diff
@@ -2,7 +2,7 @@
 node version 10
 ============================================================
 @hapi/hapi
-  17.9.0-18.4.1
+  17.9.0-20.1.5
```

Comment:
This is not true. 

Since we "skip" test by exiting if node version is low, there is no error, and since there is no error, `testeachversion` considers the version valid. Some tweaks to `versions.js` are required.